sdk/android/api/org/webrtc/HardwareVideoDecoderFactory.java

```diff
@@ -132,21 +133,25 @@ private boolean isHardwareSupported(MediaCodecInfo info, VideoCodecType type) {
         return name.startsWith(QCOM_PREFIX) || name.startsWith(INTEL_PREFIX)
             || name.startsWith(EXYNOS_PREFIX) || name.startsWith(NVIDIA_PREFIX)
             // Hisi seems to support VP8.
-            || name.startsWith(HISI_PREFIX);
+            || name.startsWith(HISI_PREFIX)
+            || vcp.isExtraHardwareSupported(name, "video/x-vnd.on2.vp8", vcp.parseWithTag(vcp.loadWithDom(extraMediaCodecFile), "Decodeds"));
```

Comment:
"Decoders" not "Decodeds".  Please also check your xml setting.

Reply:
done